app/src/routes/routesMap.js

```diff
@@ -144,20 +146,21 @@ const routesMap = {
   [PLUGINS_TAB_PAGE]: `/administrate/plugins/:pluginsTab(${INSTALLED}|${STORE})`,
 
   [PROJECT_PAGE]: {
-    path: '/:projectId',
+    path: '/:organizationSlug?/:projectKey',
```

Comment:
Why `organizationSlug` is an optional parameter?

Reply:
The main problem is, that backEnd did not send to us organizationSlug in assignedProjects.
And when app is bootstraps reduxrouter creates links for projects.
If we dont have this field in assignedProjects it breaks down our app.